ex2_hal/northern_spirit/equipment_handler/include/northern_spirit_io.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2021  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+/**
+ * @file northern_spirit_io.c
+ * @author Thomas Ganley
+ * @date 2021-10-26
+ */
+#ifndef NORTHERN_SPIRIT_IO_H
+#define NORTHERN_SPIRIT_IO_H
+
+#include "northern_spirit_handler.h"
+#include "FreeRTOS.h"
```

Comment:
Lots of unneeded includes here too. Includes not needed by the header but needed in the C file should be only in the C file